### PR TITLE
[ETLYN-38] - ფოტოების კარუსელი

### PR DESCRIPTION
**Pull request by Alsandre**
https://etlyn.atlassian.net/browse/ETLYN-38

### CODE REVIEW

**Comment by mtielitsignebi** [comment]
🎉 Nice work, all checks have passed!

File: src/pages/Home/HomePage.tsx [hunk 1]
```diff
@@ -16,7 +29,11 @@ export const HomePage = (): JSX.Element => {
         <HistorySection />
         <Divider />
         <GoalSection />
-        <Carousel />
+        <StyledCarousel
+          showSingleSlide={platform !== Platform.Web}
```
Comment (digmel) [inline src/pages/Home/HomePage.tsx:27]: On mobile, images are so large and they are breaking layout, please add more styles. Also:
```suggestion
        <StyledCarousel
          showSingleSlide={platform === Platform.Mobile}
```

**Review by digmel — CHANGES_REQUESTED** [review]

File: src/pages/Home/HomePage.tsx [hunk 2]
```diff
@@ -16,7 +29,11 @@ export const HomePage = (): JSX.Element => {
...
+          imageList={IMAGE_LIST}
```
Comment (digmel) [inline src/pages/Home/HomePage.tsx:28]: I think you should not pass IMAGE_LIST. Instead, set 'children' and keep the 'data.map' loop here. The Carousel library should animate whatever is in 'children,' whether it's an image list, a div, or cards for blogs, reviews, or other sections.

**Comment by mtielitsignebi** [comment]
🎉 Nice work, all checks have passed!

**Review by digmel — APPROVED** [review]
Nice work!

File: src/pages/Home/HomePage.tsx [hunk 3]
```diff
@@ -4,10 +4,17 @@ import { GoalSection } from "../../components/GoalSection";
...
 import { Divider } from "../../components/Divider";
+import { usePlatform } from "../../hooks";
+import { Platform } from "../../hooks/usePlatform";
+import { useControlXState } from "@etlyn/control-x";
 
 export const HomePage = (): JSX.Element => {
+//@ts-ignore
+  const IMAGE_LIST = useControlXState("gallery-core").images.map(image => ({imageURL: image.url}))
+  const platform = usePlatform();
+  console.log(IMAGE_LIST)
```
Comment (digmel) [inline src/pages/Home/HomePage.tsx:17]: Try not to keep any conosle.log in the PR.

**Pull request merged**